### PR TITLE
v3.100 mastering

### PR DESCRIPTION
**Pull request by m4rc1e**
Changes since previous release:

- Improved STAT table
- Fixed fvar instances
- Use `gftools builder` instead of shell script
- Remove post processing scripts (decompose-tranformed-components etc) since fontmake can now do this
- Add web fonts

### CODE REVIEW

**Comment by m4rc1e** [comment]
@raphlinus you're probably very busy but since this is your baby, would you mind taking a look? if not, I can get someone else.

More than happy to explain further what has changed.

**Comment by m4rc1e** [comment]
@raphlinus would you like us to review this for you?

**Comment by raphlinus** [comment]
Thanks for the ping. I took a quick skim and didn't see any problems, and was hoping to have time for a deeper review, but just at the moment I am seriously feeling the crunch (I have a paper due Monday). So please go ahead and do what you need to do to get this merged.

File: sources/config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sources:
+  - Inconsolata.glyphs
+axisOrder:
+  - wdth
+  - wght
+familyName: Inconsolata
+buildStatic: true
+flattenComponents: false
```
Comment (RosaWagner) [inline sources/config.yaml:8]: @m4rc1e why false ? 
🔥 FAIL: Check glyphs do not have components which are themselves components. ([com.google.fonts/check/glyf_nested_components](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyf_nested_components))
🔥 FAIL The following glyphs have components which themselves are component glyphs:
Aacute
Abreve
uni1EAE
uni1EAE
uni1EB6
uni1EB6
uni1EB0
uni1EB0
uni1EB2
uni1EB4 and **333 more**.

Comment (m4rc1e) [reply sources/config.yaml:8]: This family has been hinted with VTT. If we flatten these, the hinting won't be compatible. The version we currently serve also has this issue.

**Review by RosaWagner — APPROVED** [review]

**Comment by m4rc1e** [comment]
Thanks both. I'll push it to google/fonts.

**Comment by ClHg2** [comment]
@m4rc1e

> Thanks both. I'll push it to google/fonts.

I'm really sorry to disturb after 2 years, but would you mind answering this small question? This pull request is named 'v3.100', but the version in google/fonts seems to be 'v3.001'. I wonder if the version in google/fonts is up to date?

**Pull request merged**